components/script/dom/webglrenderingcontext.rs

```diff
@@ -1938,12 +2001,32 @@ impl WebGLRenderingContextMethods for WebGLRenderingContext {
     }
 
     // https://www.khronos.org/registry/webgl/specs/latest/1.0/#5.14.8
+    #[allow(unrooted_must_root)]
```

Comment:
Why is this necessary?

Reply:
Because of the dictionary iterator()
```  
Expression of type std::collections::hash_map::Iter<'_, u32, dom::webglrenderingcontext::TextureUnitBindings> must be rooted'''
```
Is there any better way to avoid that?

Reply:
Add a new entry to https://github.com/servo/servo/blob/master/components/script_plugins/unrooted_must_root.rs#L57.